### ISSUE TITLE
Extend MeCab plug-in to support morpheme n-gram

### ISSUE THREAD

**Issue by kmaehashi** [jubatus/jubatus#1054]
Currently MeCab plugin just splits the input string and use surface of each morpheme as a feature.
It is better to support morpheme n-gram to extract the context of the natural language.

I'm thinking of adding a new "ngram" parameter (which is optional and defaults to 1) to the mecab plug-in configuration as follows:

```
"string_types": {
  "mecab": {
    "method": "dynamic",
    "path": "libmecab_splitter.so",
    "function": "create",
    "arg": "-d /usr/lib64/mecab/dic/ipadic",
    "ngram": 2
  }
}
```

For instance, when `本日は晴天` is given as an input, `{"本日", "は", "晴天"}` is extracted when "ngram" is 1, whereas `{"本日|は", "は|晴天"}` is extracted when "ngram" is 2.

**Comment by kmaehashi** [comment]
Updated docs for this: https://github.com/jubatus/website/pull/236

**Comment by kmaehashi** [comment]
Fixed via #1070
